content/docs/reference/cli-ip-allow.md

```diff
@@ -0,0 +1,150 @@
+
+| Option        | Description | Type   | Required  |
+| ------------- | ----------- | ------ | :------: |
+| `--context-file` | [Context file](/docs/reference/cli-set-context#using-a-named-context-file) path and file name | string | |
+| `--project-id`  | Project ID  | string | Only if your Neon account has more than one project |
+
+#### Examples
+
+```bash
+neonctl ip-allow list
+```
+
+List the IP allowlist with the `--output` format set to `json`:
+
+```bash
+neonctl ip-allow list --output json
+```
+
+### add
+
+This subcommand allows you to add IP addresses to the IP allowlist for your Neon project.
+
+#### Usage
+
+```bash
+neonctl ip-allow add [ips ...] [options]
+```
+
+#### Options
+
+In addition to the Neon CLI [global options](/docs/reference/neon-cli#global-options), the `add` subcommand supports these options:
+
+| Option        | Description | Type   | Required  |
+| ------------- | ----------- | ------ | :------: |
+| `--context-file` | [Context file](/docs/reference/cli-set-context#using-a-named-context-file) path and file name | string | |
+| `--project-id`  | Project ID  | string | Only if your Neon account has more than one project |
+
+#### Example
+
+<CodeBlock shouldWrap>
+
+```bash
+neonctl ip-allow add 192.168.1.3
+```
+
+</CodeBlock>
+
+### remove
+
+This subcommand allows you to remove IP addresses from the IP allowlist for your project.
+
+#### Usage
+
+```bash
+neonctl ip-allow remove [ips ...] [options]
+```
+
+#### Options
+
+In addition to the Neon CLI [global options](/docs/reference/neon-cli#global-options), the `remove` subcommand supports these options:
+
+| Option        | Description | Type   | Required  |
+| ------------- | ----------- | ------ | :------: |
+| `--context-file` | [Context file](/docs/reference/cli-set-context#using-a-named-context-file) path and file name | string | |
+| `--project-id`  | Project ID  | string | Only if your Neon account has more than one project |
+
+#### Example
+
+<CodeBlock shouldWrap>
+
+```bash
+neonctl ip-allow remove 192.168.1.3
+```
+
+</CodeBlock>
+
+### reset
+
+This subcommand allows you to reset the list of IP addresses. You can reset to different IP addresses. If you specify no addresses, currently defined IP addresses are removed.
+
+#### Usage
+
+```bash
+neonctl ip-allow reset [ips ...] [options]
+```
+
+#### Options
+
+In addition to the Neon CLI [global options](/docs/reference/neon-cli#global-options), the `reset` subcommand supports these options:
+
+| Option        | Description | Type   | Required  |
+| ------------- | ----------- | ------ | :------: |
+| `--context-file` | [Context file](/docs/reference/cli-set-context#using-a-named-context-file) path and file name | string | |
+| `--project-id`  | Project ID  | string | Only if your Neon account has more than one project |
+
+#### Example
+
+<CodeBlock shouldWrap>
+
+```bash
+neonctl ip-allow reset 192.168.1.1
+```
+
+</CodeBlock>
+
+<NeedHelp/>
```

Comment:
All of the examples on `ip-allow` are missing the mention `--project-id` option/argument. The command will fail without `--project-id`. Users either have to specify the option or add it to context file.

Reply:
Added